src/main/java/org/mule/extension/email/internal/util/message/EmailMessage.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.email.internal.util.message;
+
+import static org.mule.extension.email.internal.util.EmailUtils.getMultipart;
+
+import org.mule.extension.email.api.exception.EmailException;
+import org.mule.runtime.api.metadata.MediaType;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import javax.mail.MessagingException;
+import javax.mail.Multipart;
+import javax.mail.Part;
+
+/**
+ * Abstraction to represent a email message, exposing its body text and attachments.
+ *
+ * @since 1.2.0
+ */
+public class EmailMessage {
+
+  private static final String MULTIPART_MIXED = MediaType.MULTIPART_MIXED.toRfcString();
+  private static final String MULTIPART_ALTERNATIVE = MediaType.create("multipart", "alternative").toRfcString();
```

Comment:
This MULTIPART_ALTERNATIVE is already defined somewhere else, maybe it should be defined only once and be public.